gravitee-apim-gateway/gravitee-apim-gateway-http/src/main/java/io/gravitee/gateway/reactive/http/vertx/VertxHttpServerResponse.java

```diff
@@ -95,19 +95,15 @@ public Completable end(final GenericExecutionContext ctx) {
             }
             prepareHeaders();
 
-            final AtomicReference<Subscription> subscriptionRef = new AtomicReference<>();
```

Comment:
I don't remember if we had a test to cover this. Could you verify/create a dedicated one to ensure the dispose/cancel is propagated?